docs/changelog.rst

```diff
@@ -11,6 +11,8 @@ for Python 3.6).
 
 New features:
 
+* Added option use `cover_art_arl` as an album art source in the `fetchart` plugin.
```

Comment:
Minor typo (missing "to", "arl" instead of "url") 👀

```suggestion
* Added option to use `cover_art_url` as an album art source in the `fetchart` plugin.
```